MrtCore/console_unpackaged_app/console_unpackaged_app.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <windows.h>
+#include <wil/resource.h>
+#include <iostream>
+
+#include "winrt\Windows.Foundation.h"
+#include "winrt\Windows.Foundation.Collections.h"
+#include "winrt\Microsoft.ApplicationModel.Resources.h"
+
+#include <MddBootstrap.h>
+
+using namespace winrt;
+using namespace winrt::Microsoft::ApplicationModel::Resources;
+
+HRESULT LoadProjectReunion()
+{
+    // Take a dependency on Project Reunion v0.8 preview.
+    const UINT32 majorMinorVersion{ 0x00000008 };
+    PCWSTR versionTag{ L"preview" };
+    const PACKAGE_VERSION minVersion{};
+    HRESULT hr{ MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion) };
+    if (FAILED(hr))
+    {
+        wprintf(L"Error 0x%08X in MddBootstrapInitialize(0x%08X, %s, %hu.%hu.%hu.%hu)\n",
+            hr, majorMinorVersion, versionTag, minVersion.Major, minVersion.Minor, minVersion.Build, minVersion.Revision);
+        return hr;
+    }
+
+    return S_OK;
+}
+
+int wmain(int argc, wchar_t* argv[])
+{
+    // Print usage help.
+    if ((argc < 2) || (_wcsicmp(argv[1], L"--help") == 0) || (_wcsicmp(argv[1], L"-?") == 0))
+    {
+        std::wcout << "Usage: console_unpackaged_app.exe [options] [mode]\n"
+            "options:\n"
+            "  -?, --help = Display help\n"
+            "mode:\n"
+            "  Mode for app run. Can be...\n"
+            "       Default or default\n"
+            "       Override or override\n"
+            "       Fallback or fallback\n"
+            "  Default and override retrieve a sample string from string resource files.\n"
+            "  For the override case, this sample uses the German language.\n"
+            "  Fallback corresponds to the resource-not-found case, where we fallback to a legacy resource loader.\n"
+            "\n"
+            "Examples:\n"
+            "  Get the sample string for the default resource context\n"
+            "    console_unpackaged_app.exe default\n"
+            "  Get the sample string for the override resource context (sample uses the German language for the override context)\n"
+            "    console_unpackaged_app.exe override\n"
+            "  Get the sample string for the resource-not-found fallback case\n"
+            "    console_unpackaged_app.exe fallback\n";
+        return 1;
+    }
+
+    // Initialize dynamic dependencies so we can consume the Project Reunion APIs in the Project Reunion framework package from this unpackaged app. 
+    HRESULT mddInitHr = LoadProjectReunion();
+    if (FAILED(mddInitHr))
+    {
+        return 1;

```

Comment:
Maybe add an error message?

```suggestion
        std::wcout << "Could not load Project Reunion!" << std::endl;
        return 1;
```